contrib/guix/patches/glibc-2.24-no-build-time-cxx-header-run.patch

```diff
@@ -0,0 +1,98 @@
+commit dc23a45db566095e83ff0b7a57afc87fb5ca89a1
```

Comment:
Same

Reply:
This should be the correct link: https://sourceware.org/git/?p=glibc.git;a=commit;h=fc3e1337be1c6935ab58bd13520f97a535cf70cc